aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -921,6 +930,23 @@ void TensorIterator::populate_operands(TensorIteratorConfig& config) {
   num_outputs_ = config.num_outputs_;
 }
 
+void TensorIterator::create_reduction_temporaries(){
```

Comment:
same here, don't we have single output reductions?  `create_multi_reduction_temporaries`?

Reply:
Good point